.github/ISSUE_TEMPLATE/documentation-improvement.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Documentation improvement
+about: Suggest an improvement or report a bug in the documentation
+---
+
+<!-- Thank you for your contribution. Before you submit the issue:
+1. Search open and closed issues for duplicates.
+2. Read the contributing guidelines.
+3. Assign the Documentation project.
+-->
+
+**Description**
+
+<!-- Provide a clear and concise description of the potential documentation improvement.-->
+
+**Area**
+
+<!-- Provide the area the document refers to. For example, write: 
+* Application Connector
+* Event Mesh
+* Kyma Environment Broker
+* Rafter -->
+
+**Reasons**
+
+<!-- Explain why we should improve the document. -->
+
+**Assignees**
+
+@kyma-project/technical-writers
```

Comment:
What is the process here? Do we assign all the TWs, or just the one responsible for the repository?

Just asking, perhaps this is the thing that was agreed upon in the org.

Reply:
I checked other repositories and we usually assign all the TWs. Still, it is just a notification.